src/azure-cli/azure/cli/command_modules/synapse/_params.py

```diff
@@ -128,6 +161,130 @@ def load_arguments(self, _):
+            c.argument('label_name', help='The label name.', options_list=['--label'])
+
+    # synapse sql pool tde
+    with self.argument_context('synapse sql pool tde') as c:
+        c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
+        c.argument('status', arg_type=get_enum_type(TransparentDataEncryptionStatus),
+                   required=True, help='Status of the transparent data encryption.')
+
+    # synapse sql pool threat-policy
+    with self.argument_context('synapse sql pool threat-policy') as c:
+        c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
+
+    with self.argument_context('synapse sql pool threat-policy update') as c:
+        _configure_security_or_audit_policy_storage_params(c)
+
+        policy_arg_group = 'Policy'
+        notification_arg_group = 'Notification'
+
+        c.argument('state',
+                   arg_group=policy_arg_group,
+                   help='Threat detection policy state',
+                   arg_type=get_enum_type(SecurityAlertPolicyState))
+
+        c.argument('retention_days',
+                   arg_group=policy_arg_group,
+                   help='The number of days to retain threat detection logs.')
+
+        c.argument('disabled_alerts',
+                   arg_group=policy_arg_group,
+                   help='List of disabled alerts.',
+                   nargs='+')
+
+        c.argument('email_addresses',
+                   arg_group=notification_arg_group,
+                   help='List of email addresses that alerts are sent to.',
+                   nargs='+')
+
+        c.argument('email_account_admins',
+                   arg_group=notification_arg_group,
+                   help='Whether the alert is sent to the account administrators.',
+                   arg_type=get_three_state_flag())
+
+    # synapse sql pool audit-policy
+    with self.argument_context('synapse sql pool audit-policy') as c:
+        c.argument('sql_pool_name', arg_type=name_type, id_part='child_name_1', help='The SQL pool name.')
+
+    with self.argument_context('synapse sql pool audit-policy update') as c:
+        _configure_security_or_audit_policy_storage_params(c)
+        c.argument('storage_account_subscription_id', arg_group=storage_arg_group,
+                   options_list=['--storage-subscription'],
+                   help='The subscription id of storage account')
+        c.argument('is_storage_secondary_key_in_use', arg_group=storage_arg_group,
+                   arg_type=get_three_state_flag(), options_list=['--use-secondary-key'],
+                   help='Indicates whether using the secondary storeage key or not')
+
+        c.argument('is_azure_monitor_target_enabled', options_list=['--enable-azure-monitor'],
+                   help='Whether enabling azure monitor target or not.',
+                   arg_type=get_three_state_flag())
+
+        policy_arg_group = 'Policy'
```

Comment:
>policy_arg_group = 'Policy [](start = 8, length = 26)

Duplicate definition
+230 +274

Reply:
fixed

---
In reply to: [533229192](https://github.com/Azure/azure-cli/pull/16095#discussion_r533229192) [](ancestors = 533229192)